Documentation/ContentObjects/Fluidtemplate/Index.rst

```diff
@@ -17,6 +17,140 @@ It can be used in :ref:`content elements <t3coreapi:adding-your-own-content-elem
+
+..  code-block:: typoscript
+    :caption: Before migration, EXT:my_sitepackage/Configuration/TypoScript/setup.typoscript
+
+    # Todo: migrate to PAGEVIEW
+    page = PAGE
+    page {
+        10 = FLUIDTEMPLATE
+        10 {
+            templateName = TEXT
+            templateName {
+                stdWrap {
+                    cObject = TEXT
+                    cObject {
+                        data = levelfield:-2, backend_layout_next_level, slide
+                        override {
+                            field = backend_layout
+                        }
+                        split {
+                            token = pagets__
+                            1 {
+                                current = 1
+                                wrap = |
+                            }
+                        }
+                    }
+                    ifEmpty = Standard
+                }
+            }
+
+            templateRootPaths {
+                100 = EXT:my_sitepackage/Resources/Private/Templates/Pages/
+            }
+
+            partialRootPaths {
+                100 = EXT:my_sitepackage/Resources/Private/Partials/Pages/
+            }
+
+            layoutRootPaths {
+                100 = EXT:my_sitepackage/Resources/Private/Layouts/Pages/
+            }
+
+            variables {
+                pageUid = TEXT
+                pageUid.data = page:uid
+
+                pageTitle = TEXT
+                pageTitle.data = page:title
+
+                pageSubtitle = TEXT
+                pageSubtitle.data = page:subtitle
+
+                parentPageTitle = TEXT
+                parentPageTitle.data = levelfield:-1:title
+            }
+
+            dataProcessing {
+                10 = menu
+                10.as = mainMenu
+            }
+        }
+    }
+
+.. code-block:: typoscript
+    :caption: After migration, EXT:my_sitepackage/Configuration/TypoScript/setup.typoscript
+
+    page = PAGE
+    page {
+        10 = PAGEVIEW
+        10 {
+            paths {
+                100 = EXT:my_sitepackage/Resources/Private/Templates/
+            }
+            variables {
+                parentPageTitle = TEXT
+                parentPageTitle.data = levelfield:-1:title
+            }
+            dataProcessing {
+                10 = menu
+                10.as = mainMenu
+            }
+        }
+    }
+
+In Fluid, the pageUid is available as :html:`{page.uid}` and pageTitle
+as :html:`{page.title}`, the subtitle with :html:`{page.subtitle}`.
+
+In this example some Fluid templates have to be moved. Move:
+
+:path:`EXT:my_sitepackage/Resources/Private/Templates/Pages/`
+    The page Templates can stay in this folder.
+:path:`EXT:my_sitepackage/Resources/Private/Partials/Pages/`
+    Move files to :path:`EXT:my_sitepackage/Resources/Private/Templates/Partials/`
+:path:`EXT:my_sitepackage/Resources/Private/Layouts/Pages/`
+    Move files to :path:`EXT:my_sitepackage/Resources/Private/Templates/Layouts/`
```

Comment:
Could we maybe format those tasks into a bullet point list?

Reply:
Why not a definition list as we have now?